token-group/interface/Cargo.toml

```diff
@@ -9,12 +9,20 @@ edition = "2021"
 
 [dependencies]
 bytemuck = "1.19.0"
+num-derive = "0.4"
+num-traits = "0.2"
+solana-decode-error = "2.1.0"
+solana-instruction = "2.1.0"
+solana-msg = "2.1.0"
 solana-program = "2.1.0"
```

Comment:
It looks the only usage of solana-program can be removed from state.rs, see my other comment